test/e2e/common/security_context.go

```diff
@@ -180,9 +177,12 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			podClient.WaitForSuccess(podName, framework.PodStartTimeout)
 			return podName
 		}
-
-		It("should run the container as unprivileged when false [LinuxOnly] [NodeConformance]", func() {
-			// This test is marked LinuxOnly since it runs a Linux-specific command, and Windows does not support Windows escalation.
+		/*
+			Release : v1.14
+			Testname: Security Context, readOnlyRootFilesystem=false.
+			Description: Create a container to run in unprivileged mode by configuring pod's SecurityContext Privileged option as false. Pod MUST be in Succeeded phase.
+		*/
+		framework.ConformanceIt("should run the container as unprivileged when false [NodeConformance]", func() {
```

Comment:
Please do not remove the ``[LinuxOnly]`` label, test contains a SecurityContext, which is not working on Windows.